config.go

```diff
@@ -35,6 +35,7 @@ const (
 	defaultImagePath            = "/usr/share/clear-containers/clear-containers.img"
 	defaultHypervisorPath       = "/usr/bin/qemu-lite-system-x86_64"
 	defaultProxyURL             = "unix:///run/cc-oci-runtime/proxy.sock"
+	defaultPauseBinPath         = "/tmp/bundles/pause_bundle/rootfs/bin/pause"
```

Comment:
@jodh-intel I can do this change, I wanted to do that anyway.
@sameo should I add a default binary pause path or we can consider we have to read it from the config file ? In case you want a default one, what do you think as a default location ?